public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/react-table-config.d.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  UseColumnOrderInstanceProps,
```

Comment:
Why do we need this file? I don't see it in PMM. 🤔 And if we really need it this is probably not the best place to put it.

Reply:
because I think in PMM there's a workaround to a problem with `column.getSortByToggleProps`, casting directly to TableOptions (see [here](https://github.com/percona/grafana-dashboards/blob/PMM-2.0/pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx#L89-L90)). It's still possible to do that actually, but I don't really know which I prefer. Maybe casting is easier though 🤷‍♂️

Reply:
removed it